### PR TITLE
Refresh contribute page

### PR DESCRIPTION
**Pull request by carolynvs**
# What does this change
* Incorporate feedback on how we link to issue labels
* Link to list of mixins and mixin ideas
* Offer mentoring
* Link to quickstart

Preview at https://deploy-preview-1121--porter.netlify.app/contribute/

# What issue does it fix
N/A

# Notes for the reviewer
🍫

# Checklist
- [ ] Unit Tests
- [x] Documentation
- [ ] Schema (porter.yaml)

### CODE REVIEW

File: docs/content/contribute.md [hunk 1]
```diff
@@ -77,6 +73,12 @@ do:
 * Project management or other skillsets would be amazing as well! Contact
   Carolyn and let's coordiante. 🙌
```
Comment (iennae) [inline docs/content/contribute.md:74]: I know it's been around but coordinate

File: docs/content/contribute.md [hunk 2]
```diff
@@ -86,14 +88,17 @@ reach out to a maintainer on [Slack][slack].
 [skeletor]: https://github.com/deislabs/porter-skeletor
 [mixin-dev-guide]: /mixin-dev-guide/
 [roadmap]: /roadmap
+[existing-mixins]: https://github.com/deislabs/porter-packages/blob/main/mixins/index.json
+[mixin-ideas]: https://github.com/deislabs/porter/issues?q=is%3Aissue+is%3Aopen+label%3A%22mixin+idea%22
+[twitter]: https://github.com/
```
Comment (iennae) [inline docs/content/contribute.md:93]: This twitter link looks like it's dangling.

Comment (carolynvs) [reply docs/content/contribute.md:93]: oops! thanks yes it's not used anywhere

**Comment by carolynvs** [comment]
@iennae I've updated the PR with your suggested changes, please take another look.

**Pull request merged**